packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -193,6 +193,10 @@ class HeaderMenu extends React.Component {
       ...rest
     } = this.props;
 
+    const hasActiveChildren =
+      React.Children.map(children, (child) => {
+        return child.props.isActive || child.props.isCurrentPage;
+      }).length > 0;
```

Comment:
slight performance improvement. you might have to do `.toArray()` before calling `.some`, not sure
```suggestion
    const hasActiveChildren =
      React.Children.some(children, (child) => {
        return child.props.isActive || child.props.isCurrentPage;
      });
```

Reply:
Updated 👍